docs/vagrant-spk/customizing.md

```diff
@@ -94,7 +94,7 @@ Other things you probably want to do in this script include:
 
 - Building folder structures in `/var`.  `/var` is the only non-tmpfs folder mounted R/W, and when a grain is first launched, it will start out empty.  It will persist between runs of the same grain, but be unique per app instance.
 - Preparing a database and running migrations.  You can also manually generate some tables once, place them somewhere under `/opt/app`, and copy them to `/var/lib/mysql` if your app takes a while to do migrations, at the potential cost of producing a larger `.spk`.
-- Launching other daemons that your app uses (`mysqld`, `redis-server`, `php5-fpm`, `uwsgi`, etc.)
+- Launching other daemons that your app uses (`mysqld`, `redis-server`, `php7-fpm`, `uwsgi`, etc.)
```

Comment:
Maybe it would be better to just say `php-fpm`, to make this a bit more evergreen? The executable name varies from distro to distro and I don't think it's that important that it mach exactly what's currently on debian -- especially since the lesp/lemp stacks are going to have that as an example anyway.

Reply:
I agree here. I will plan to force push this before merging, as a commit to delete a digit would feel sad.